reminderDialog.zsh

```diff
@@ -299,45 +473,77 @@ function updateRequiredVariables() {
     # IT Support Variables
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-    supportTeamName="IT Support"
-    supportTeamPhone="+1 (801) 555-1212"
-    supportTeamEmail="rescue@domain.org"
-    supportTeamWebsite="https://support.domain.org"
-    supportTeamHyperlink="[${supportTeamWebsite}](${supportTeamWebsite})"
-    supportKB="KB8675309"
-    infobuttonaction="https://servicenow.domain.org/support?id=kb_article_view&sysparm_article=${supportKB}"
-    supportKBURL="[${supportKB}](${infobuttonaction})"
+    local defaultSupportTeamName="${supportTeamName:-"IT Support"}"
+    setPreferenceValue "supportTeamName" "${supportTeamName_managed}" "${supportTeamName_local}" "${defaultSupportTeamName}"
+
+    local defaultSupportTeamPhone="${supportTeamPhone:-"+1 (801) 555-1212"}"
+    setPreferenceValue "supportTeamPhone" "${supportTeamPhone_managed}" "${supportTeamPhone_local}" "${defaultSupportTeamPhone}"
+
+    local defaultSupportTeamEmail="${supportTeamEmail:-"rescue@domain.org"}"
+    setPreferenceValue "supportTeamEmail" "${supportTeamEmail_managed}" "${supportTeamEmail_local}" "${defaultSupportTeamEmail}"
+
+    local defaultSupportTeamWebsite="${supportTeamWebsite:-"https://support.domain.org"}"
+    setPreferenceValue "supportTeamWebsite" "${supportTeamWebsite_managed}" "${supportTeamWebsite_local}" "${defaultSupportTeamWebsite}"
+
+    local defaultSupportKB="${supportKB:-"KB8675309"}"
+    setPreferenceValue "supportKB" "${supportKB_managed}" "${supportKB_local}" "${defaultSupportKB}"
+
+    local defaultInfobuttonaction="https://servicenow.domain.org/support?id=kb_article_view&sysparm_article=${supportKB}"
+    setPreferenceValue "infobuttonaction" "${infobuttonaction_managed}" "${infobuttonaction_local}" "${defaultInfobuttonaction}"
+
+    local defaultSupportKBURL="[${supportKB}](${infobuttonaction})"
+    setPreferenceValue "supportKBURL" "${supportKBURL_managed}" "${supportKBURL_local}" "${defaultSupportKBURL}"
 
 
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
     # Title, Message and  Button Variables
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-    title="macOS ${titleMessageUpdateOrUpgrade} Required"
```

Comment:
I'm not seeing `titleMessageUpdateOrUpgrade` anywhere in your version of  [`org.churchofjesuschrist.dorm.plist`](https://github.com/Acne-Studios/AS_DDMOSReminder/blob/a03564fd3b48bd9238b82b9c2a91e843d26dd958/org.churchofjesuschrist.dorm.plist)